src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/FormTheme/product.html.twig

```diff
@@ -52,6 +52,28 @@
   </li>
 {% endblock %}
 
+{% block packed_product_row %}
+  <li class="packed-product entity-item" id="{{ form.vars.name }}">
```

Comment:
```suggestion
  <li class="packed-product entity-item" id="packed-product-{{ form.vars.name }}">
```

Reply:
We can't leave ids as it is: 0, 1, x... It makes no sense

Reply:
This is how the collection component works, it's a list of sub forms which need to be indexed by an ID or a 0-indexed That's the core behaviour of the the Symfont form type so it's fine for me 🤔

Reply:
> This is how the collection component works, it's a list of sub forms which need to be indexed by an ID or a 0-indexed That's the core behaviour of the the Symfont form type so it's fine for me 🤔

Really, symfony contains some components with an id as an int only? That's really weird

Reply:
Oh and by the way, last precision, it's only `form.vars.name` which value is an int, it's not the full name nor the full ID
The full ID automatically generated by Symfony would follow the whole form structure and would look like `product_stock_packed_products_1` similar to the full name which would be `product[stock][packed_products][1]`

Reply:
I'm of course confused about adding `0`, `1`, `2` inside an `id` HTML attribute, it doesn't sound like it's unique at all and could lead to an invalid HTML structure according to W3C specs

Reply:
Its not like we have <input id="0">, but it will be like  <input id="product_stock_packed_products_0">  as Joe mentioned. I dont see any problem with that.

Reply:
```suggestion
  <li class="packed-product entity-item" id="packed-product-{{ form.vars.name }}">
```

My bad I was completely mistaken!! I had in mind that Valentin's suggestion was actually used so I didn't understand but indeed @Amoifr we need to prefix the ID here or will end up with `id="1"` `id="2"` and so on

Reply:
attribute removed